### DISCUSSION TITLE
Improved type annotations in version 0.104.0

### DISCUSSION THREAD

**Discussion by dantownsend** [piccolo-orm/piccolo#748] [Show and tell]
We made some big improvements in version `0.104.0` to Piccolo's type annotations.

This means it will work even better with MyPy and Pylance.

I posted a couple of blog articles explaining how we did this:

 * https://piccolo-orm.com/blog/testing-python-type-annotations/
 * https://piccolo-orm.com/blog/advanced-type-annotations-using-python-s-type-var/

**Comment by v3ss0n** [comment]
Keep it up!  Picoccolo is The most pythonic ORM , an ORM that python deserves.
